Turtle/WithSensorData/SensorDataExample14.ttl

```diff
@@ -0,0 +1,55 @@
+@prefix epcis: <https://ns.gs1.org/epcis/> .
+@prefix gs1:   <https://gs1.org/voc/> .
+@prefix owl:   <http://www.w3.org/2002/07/owl#> .
+@prefix vtype: <urn:epcglobal:epcis:vtype:> .
+@prefix example-mda: <http://epcis.example.com/mda/> .
+@prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix example2: <http://epcis.example.com/ns/> .
+@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix cbv:   <https://ns.gs1.org/cbv/> .
+@prefix md:    <https://ns.gs1.org/epcismd/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix example1: <http://ns.example.com/epcis/> .
+@prefix cbvmda: <urn:epcglobal:cbv:mda:> .
+
+<ni:///sha-256;77c5b495d391767b1d3b8e7c669c9fee3965a778a9bfe793e85602c971b98436?ver=CBV2.0>
+    a                          epcis:ObjectEvent ;
+    epcis:action               "OBSERVE" ;
+    epcis:bizStep              <urn:epcglobal:cbv:bizstep:sensor_reporting> ;
+    epcis:epcList              <urn:epc:id:sgtin:4012345.022222.1234> ;
+    epcis:eventTime            "2021-05-18T12:23:34+00:00"^^xsd:dateTime ;
+    epcis:eventTimeZoneOffset  "+01:00" ;
+    epcis:readPoint            <urn:epc:id:sgln:4012345.00005.0> ;
+    epcis:sensorElementList    [ epcis:sensorMetadata  [ epcis:time "2021-05-20T12:23:34+00:00"^^xsd:dateTime
+                                                       ] ;
+                                 epcis:sensorReport    [ epcis:type        'Angle' ;
```

Comment:
regenerate Turtle from JSON.
- use consistent quotes `"` not `'`
- datatype `^^xsd:double` is missing
- `epcis:type` is wrong, should be `epcis:measurementType`